editor/editor_properties.h

```diff
@@ -279,7 +279,44 @@ class EditorPropertyFlags : public EditorProperty {
 	EditorPropertyFlags();
 };
 
-class EditorPropertyLayersGrid;
+///////////////////// LAYERS /////////////////////////
+
+class EditorPropertyLayersGrid : public Control {
+	GDCLASS(EditorPropertyLayersGrid, Control);
+
+private:
+	Vector<Rect2> flag_rects;
+	Rect2 expand_rect;
+	bool expand_hovered = false;
+	bool expanded = false;
+	int expansion_rows = 0;
+	int hovered_index = -1;
+	bool read_only = false;
+	int renamed_layer_index = -1;
+	PopupMenu *layer_rename;
+	ConfirmationDialog *rename_dialog;
+	LineEdit *rename_dialog_text;
+
+	void _rename_pressed(const int p_menu);
```

Comment:
`const` is overkill for `int` parameters (at least that's our current consensus :))
```suggestion
	void _rename_pressed(int p_menu);
```